docs/sdk-and-tools/sdk-py/mxpy-cli.md

```diff
@@ -262,7 +301,47 @@ Using the `--function` argument we specify the function we want to call and with
 
 [comment]: # (mx-context-auto)
 
-## Querying a Smart Contract
+### Calling the smart contract providing the ABI file
+
+Same as we did for deploying the contract, we can call functions by providing the ABI file and the arguments file.
+
+Since we deployed the [multisig contract](https://github.com/multiversx/mx-contracts-rs/tree/main/contracts/multisig), we'll call the `proposeTransferExecute` endpoint.
+
+First, we'll prepare the file containing the endpoints arguments. We'll refer to this file as `call_multisig_arguments.json`. The `proposeTransferExecute` endpoint requires four arguments, the first is of type `Address`, the second one is of type `BigUInt`, the third is of type `Option<u64>` and the fourth is of type `variadic<bytes>`. All the arguments in this file **should** be placed inside a list. The arguments file should look like this:
+
+```json
+[
+  {
+    "bech32": "erd1qqqqqqqqqqqqqpgqs63rcpahnwtjnedj5y6uuqh096nzf75gczpsc4fgtu"
+  },
+  1000000000000000000,
+  5000000,
+  [
+    {
+      "hex": "616464403037"
+    }
+  ]
+]
+```
+
+Let's go a bit through our file and see why it looks like this. First, as mentioned above, we have to place all the arguments inside a list. Then, the contract expects an address, so we provide the `bech32` representation. After that, we have a `BigUInt` value that we can provide as a number. The third value is `Option<u64>` so we provide it as a number, as well. In case we wanted to skip this value, we could've simply used `0`. The last parameter is of type `variadic<bytes>`. Because it's a variadic value we have to place the arguments inside a list and since we can't write bytes we `hex encode` the value and place it in a dictionary containing the key-value pair `"hex": "<hex_string>"`, same as we did above for the address.
```

Comment:
Some commas might help with readability:
- After `The third value is Option<u64>`
- After `Because it's a variadic value`
- After `since we can't write bytes`

Also, it might be easier to read if we split the long sentence in two: "Because it's a variadic value, we have to place the arguments inside a list. Since we can't write bytes..."

Reply:
done